### PR TITLE
Add new person-vehicle-bike detection models 

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**
Added person-vehicle-bike-detection-2003 and person-vehicle-bike-detection-2004 models as a release candidates for OpenVINO 2021.3.

### CODE REVIEW

File: models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+| MParams                         | 1.95                                      |
+| Source framework                | PyTorch\*                                 |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Inputs
+
+Name: `input`, shape: [1x3x256x448] - An input image in the format [BxCxHxW],
```
Comment (Anna-Mironova) [inline models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md:27]:
```suggestion
Name: `input`, shape: `1, 3, 256, 448` - An input image in the format `B, C, H, W`,
```

File: models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- B - batch size
+- C - number of channels
+- H - image height
+- W - image width
+
+Expected color order is BGR.
+
+## Outputs
+
+1. The `boxes` is a blob with shape: [N, 5], where N is the number of detected
```
Comment (Anna-Mironova) [inline models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md:39]:
```suggestion
1. The `boxes` is a blob with shape: `N, 5`, where N is the number of detected
```

File: models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+   bounding boxes. For each detection, the description has the format:
+   [`x_min`, `y_min`, `x_max`, `y_max`, `conf`],
+   where:
+    - (`x_min`, `y_min`) - coordinates of the top left bounding box corner
+    - (`x_max`, `y_max`) - coordinates of the bottom right bounding box corner.
+    - `conf` - confidence for the predicted class
+2. The `labels` is a blob with shape: [N], where N is the number of detected
```
Comment (Anna-Mironova) [inline models/intel/person-vehicle-bike-detection-2004/description/person-vehicle-bike-detection-2004.md:46]:
```suggestion
2. The `labels` is a blob with shape: `N`, where N is the number of detected
```

**Comment by eaidova** [comment]
Jenkins please retry a build

**Review by eaidova — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
if new models should work with old demos, please add models to relevant demo's models.lst and create test cases for them (but leave them commented out). We will uncomment when new models will be available to downloader.

**Comment by GalyaZalesskaya** [comment]
@eaidova Thank you for your patience, I had to do a huge investigation, but I fixed the drop.
@vladimir-dudnik I added unit test case and commented it. Also, I checked that the rule to pick relevant models in model.lst already covers new models, so there are no changes.

**Comment by vladimir-dudnik** [comment]
please see pre-commit checks failed

13:12:07  ./models/intel/person-vehicle-bike-detection-2004/accuracy-check.yml
13:12:07    19:14     error    too many spaces after hyphen  (hyphens)
13:12:07  
13:12:07  ./models/intel/person-vehicle-bike-detection-2003/accuracy-check.yml
13:12:07    19:14     error    too many spaces after hyphen  (hyphens)

File: models/intel/person-vehicle-bike-detection-2003/accuracy-check.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+models:
+  - name: person-vehicle-bike-detection-2003
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
```
Comment (eaidova) [inline models/intel/person-vehicle-bike-detection-2003/accuracy-check.yml:6]: please remove device

**Review by vladimir-dudnik — APPROVED** [review]

**Review by Anna-Mironova — APPROVED** [review]

**Comment by eaidova** [comment]
Jenkins please retry a build

**Comment by xonobo** [comment]
Cannot download 2003 and 2004 models. 
`python downloader.py --print_all`
command does not list these models.

**Pull request merged**